radeon-profile/gpu.cpp

```diff
@@ -131,7 +131,7 @@ void gpu::defineAvailableDataContainer() {
 
     GPUPwmStruct tmpPWm = driverHandler->getPwmSpeed();
 
-    if (tmpPWm.pwmSpeed != -1)
+    if (tmpPWm.pwmSpeed != -1 && gpuParams.pwmMaxSpeed != -1)
```

Comment:
Line 269 is necessary. I use a fanless card, and removing that line the fan speed indicator is shown, while it should be hidden.